typescript/test/test.ts

```diff
@@ -0,0 +1,11 @@
+import test from 'ava';
+import moduleName from '.';
+
+test('title', t => {
+	const err = t.throws(() => {
+		moduleName(123);
+	}, TypeError);
```

Comment:
Use the object form of the second argument here. Then you don't have to return the error. https://github.com/avajs/ava/blob/master/docs/03-assertions.md#throwsfn-expected-message